packages/model-viewer/src/test/features/annotation-spec.ts

```diff
@@ -96,6 +96,22 @@ suite('Annotation', () => {
       expect(sceneContainsHotspot(scene, hotspot)).to.be.true;
     });
 
+    test('querying it returns valid data', () => {
+      const hotspot = element.queryHotspot('hotspot-1');
+
+      expect(hotspot?.position.toString()).to.equal(toVector3D(new Vector3(1, 1, 1)).toString());
+      expect(hotspot?.normal.toString()).to.equal(toVector3D(new Vector3(0, 0, -1)).toString());
+      expect(hotspot?.facingCamera).to.be.true;
+
+      // Testing the screen space coordinate is contingent on lots of default values
+      // (canvas size, camera position and camera fov).
+      // Just test to see that it's valid.
```

Comment:
How about a simple symmetric case? If you use a cube centered at the origin and set `phi = 90deg`, then a hotspot at the origin should show up in the center of the viewport, right?

Reply:
Ah, yeah that works! Done